packages/get-platform/src/__tests__/getPlatform.test.ts

```diff
@@ -1,4 +1,5 @@
 import stripAnsi from 'strip-ansi'
+import { C } from 'ts-toolbelt'
```

Comment:
unused import

Reply:
How the heck did this end up here? 😅
Thanks for spotting this